### ISSUE TITLE
Support multiple domains (domain1.com, domain2.org, example.ru)

### ISSUE THREAD

**Issue by korney4eg** [nginx-le/nginx-le#27]
It would be nice to have support for different domains, not only FQDN's.

For example, I would like to host on single instance 2 static sites (myblog.com, otherside.ru) and 1 dynamic site (myjenkins.org) as a reverse proxy.

Does it make sence?

**Comment by paskal** [comment]
Could you please show an example of such configuration, what you expect to work and is not working at the moment?

**Comment by umputun** [comment]
I don't really understand what the issue is. Multiple FQDNs (i.e. domain names) are supported right now and you can do `LE_FQDN=myblog.com,otherside.ru,myjenkins.org`

**Comment by korney4eg** [comment]
Hey @paskal , @umputun . Thank you for quick answering.
Here is complete example:
I have static such sites:
**blog.com**(static) - files located in `/opt/sites/blog.com`
**otherside.ru**(static) - files located in `/opt/sites/otherside.ru`
**myjenkins.org**(dynamic) -  proxy `localhost:8080`

So I would like to be able to have one script with different `service*.conf` files that can handle 3 this sites.

Right now I see that it doesn't work for single static site, because on `le.sh` ([/usr/share/nginx/html path is hardcoded](https://github.com/umputun/nginx-le/blob/master/script/le.sh#L33)

Right now I'm thinking how to fix this. Maybe you can share your thoughts what approach would be better.

**Comment by dev4day** [comment]
Привет @korney4eg !
После того как сам @umputun написал, что он реально не понимает Вашего вопроса, зачем свой вопрос ставить еще более не корректно.
Лично мое мнение, уважаемый @korney4eg, такое: Вам надо углубиться в теорию. А именно:
1. Познакомиться с принципом работы LE в общем и certbot в частности
2. Ознакомиться с мапингом директорий в докере
3. Изучить хотя бы основы конфигурирование нджинкса, уделить внимание server{...}

Далее наложить одни знания на другие и Вы поймете, что это issue ....

**Comment by umputun** [comment]
I don't see how nginx-le prevents you from using any custom configuration you like to achieve your goal. `le.sh` line you linked is not about static root for all sites, but le specific.

generally speaking, you should do 3 things:
1. map your `/opt/sites/blog.com` and `/opt/sites/otherside.ru` with compose's `volume`
1. make nginx config or multiple configs, up to you
1. for each static site and proxy define named `server` section in that config (see nginx docs) with `root` (for static sites) and proxy_pass location for the proxy.

see also [service-example.conf](https://github.com/umputun/nginx-le/blob/master/etc/service-example.conf) may give you some ideas

**Comment by korney4eg** [comment]
Ok, I got it now!

I was trying to run nginx-le container on port 8880 and 8443 in parallel with normal nginx that was running on ports 80 and 443. Of course it failed to verify certificates, because normal nginx didn't know anything about the domains and couldn't provide proper files.

After I have stopped normal nginx and added mapping to port 80 certificates were successfully created.

Will create PR with example.

**P.S.:** @dev4day thanks for RTFM
